docs/Chart.md

```diff
@@ -0,0 +1,7 @@
+# TRUST_CLAIM
```

Comment:
hey @romeoscript no no, every time a user makes a claim it is a raw claim

Its TOTALLY OK if its the same as someone else's claim

If they are verifying a claim they would do that by clicking on it

Please do NOT try to unique claims.